code/gui.py

```diff
@@ -0,0 +1,240 @@
+import os
+import tkinter as tk
+from clusterer import do_clustering
+from subprocess import Popen, PIPE
+from tkinter import messagebox
+from tkinter.filedialog import askopenfilename, askdirectory
+from VectorModelBuilder import VectorModelBuilder
+
+VMB_ARGS = (
+    ("Dataset", ""), 
+    ("Count Method", "ngram"), 
+    ("Weighting", "ppmi"), 
+    ("Output Directory", "../vector_data/"), 
+    ("Output File Name", "None"), 
+    ("Value of N", "3")
+)
+CLUSTERER_ARGS = (
+    ("Input Data File", ""), 
+    ("Output File Directory", "../vector_data/"), 
+    ("Output File Name", "classes.txt"), 
+    ("V Scalar", "1"), 
+    ("Constrain Partition", "False"), 
+    ("Constrain PCS", "False")
```

Comment:
Can you change the defaults for both these "Constrain..." options to `"True"`? That's the default in the command line version.